rpm/nano.spec

```diff
@@ -1,12 +1,13 @@
 Name:       nano
 Summary:    A small text editor
-Version:    2.8.5
+Version:    6.4
 Release:    1
 License:    GPLv3
 URL:        http://www.nano-editor.org
 Source0:    %{name}-%{version}.tar.bz2
 Patch0:     0001-autogen-Do-not-fetch-sources-at-the-build-time.patch
-BuildRequires:  pkgconfig(ncurses)
+Patch1:     0002-Build-with-older-makeinfo-that-doesn-t-support-c.patch
+BuildRequires:  pkgconfig(ncursesw)
```

Comment:
You removed some white space here, intentional?

Reply:
Not spotting any whitespace difference.